frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/validation/AuthAppCodeProcessorTest.java

```diff
@@ -159,6 +160,23 @@ void returnsCorrectErrorWhenAuthCodeIsInvalid(JourneyType journeyType, String au
         assertEquals(Optional.of(ErrorResponse.ERROR_1043), authAppCodeProcessor.validateCode());
     }
 
+    @Test
+    void IncrementsWithCorrectTTLWhenAuthCodeIsInvalidInCreationJourney() {
```

Comment:
I suggest camelCase the name of this test to `incrementsWithCorrectTTLWhenAuthCodeIsInvalidInCreationJourney`

Reply:
Nice spot, agreed!